src/regex.py

```diff
@@ -73,22 +73,24 @@ def literal(self, string):
 
         return self
 
-    def orex_or(self, string):
+    def orex_or(self, *patterns):
 
-        if isinstance(pattern, str):
-            self.expr = "(" + self.expr + "|" + pattern + ")"
+        for pattern in patterns:
+            if isinstance(pattern, str):
+                self.expr = "(" + self.expr + "|" + pattern + ")"
```

Comment:
i think the brackets need to be added at the ends only

Reply:
Removed the brackets, which I can use with a further generalisation I was exploring. 

If we decide to merge [your changes](https://github.com/HCelion/orex/pull/4), then we can generalise this even further by calling _instancer.